crates/ruff_linter/src/rules/pydocstyle/rules/indent.rs

```diff
@@ -291,10 +291,21 @@ pub(crate) fn indent(checker: &mut Checker, docstring: &Docstring) {
             let line_indent = leading_space(last);
             let line_indent_size = line_indent.chars().count();
             if line_indent_size > docstring_indent_size {
+                let mut offset_len = line_indent.text_len();
+                // When a non-space character is on the last line, the trailing quotes are also on the same line.
+                // No diagnostic is added if the indentation difference equals the indent size.
+                if last.chars().any(|c| !c.is_whitespace()) {
```

Comment:
I think we can simplify this by testing if the `line_indent.text_len() == `last.text_len()`. If the condition is true, then the last line is either all whitespace or empty